### DISCUSSION TITLE
Scalar Mixer Measurement

### DISCUSSION THREAD

**Discussion by hennryns** [jankae/LibreVNA#268] [General]
Hi @jankae , would LibreVNA be capable to do Scalar Mixer Measurement?

**Comment by jankae** [comment]
No, this is not supported. I thought about it a while ago but figured the required additional hardware for the setup and calibration would prevent most users from using it anyway.
As far as I know, you need at least a second RF source (for the LO of the mixer) and a power meter (for amplitude calibration). The power meter should also be at least compatible with the VNA (meaning the VNA can access it directly, otherwise you would need to manually calibrate a lot of points). I do not own any power meter so far and I believe that is probably the case for most LibreVNA users.
